### ISSUE TITLE
assign_editor responder with "if" conditional on issue template

### ISSUE THREAD

**Issue by mpadge** [ropensci-org/buffy#24]
This is slightly different to the conditional welcome responder of #23. We have a few different values for `submission-type` variables which we need to build some conditional logic around, primarily to invoke two different response messages dependent on these variables. Something like:

``` rb
if:
  value_matches:
    submission-type: "Standard|Estándar" # Can we just grep-or values like that?
   message: "First message"
if:
  value_matches:
    submission-type: "Stats"
   message: "Second message"
```

And then it's probably good to add
``` rb
if:
  value_matches:
    submission-type: "Pre-submission|pre-envio"
   message: "Reject message"
```

@xuanxu  Do each of those need a separate responder, or can all three `if` clauses be combined within a single responder?

**Comment by maelle** [comment]
@xuanxu @mpadge what's needed here?

**Comment by mpadge** [comment]
Ability for the command `@ropensci-review-bot assign <them> as editor` to respond with different messages dependent on a template variable. (Specifically Spanish and English messages for those respective templates.)

**Comment by maelle** [comment]
@mpadge is this only for this command? Or potentially more?

**Comment by mpadge** [comment]
It will also be generally useful in `approve` responders for the same reasons, as well as in `approve` responses to Stats submissions, which need to be different from standard approve responses. So yes, definitely more.

**Comment by xuanxu** [comment]
This require to have system in place to have multilingual templates/messages. 
Issue created upstream: https://github.com/openjournals/buffy/issues/57

**Comment by mpadge** [comment]
@xuanxu any progress here? Absence of this ability is currently leading to a fair bit of failure-prone manual work, so would be good to have implemented, if possible. Thanks

**Comment by xuanxu** [comment]
> any progress here?

Not really, sorry. There have been too many things with higher priority. I'll see if I can fit it in sometime soon, but can't make any promises right now.

**Comment by mpadge** [comment]
@xuanxu yet another ping on this issue. We have now started receiving and processing Spanish-language submissions, and it would be great to have the bot also reply in Spanish. Any possibility of this being implemented soon?
